### PR TITLE
Add debug logging to getTrackForJukebox

### PR DESCRIPTION
**Pull request by markspolakovs**

### CODE REVIEW

File: src/Classes/iTones/iTones_Utils.php [hunk 1]
```diff
@@ -357,4 +375,13 @@ public static function telnetEnd()
             self::$telnet_handle = null;
         }
     }
+
+    private static function log(...$fields)
```
Comment (mstratford) [inline src/Classes/iTones/iTones_Utils.php]: Ideally this would have a config var for turning it on / off and a logging location.

**Comment by mstratford** [comment]
Do we still want / need this?

**Comment by markspolakovs** [comment]
Nah. Bin.

**Comment by markspolakovs** [comment]
(I'll do a proper logging PR at some point, but not right now.)

**Pull request closed without merging**